airflow/www/views.py

```diff
@@ -702,6 +702,22 @@ def _iter_parsed_moved_data_table_names():
                 # Second segment is a version marker that we don't need to show.
                 yield segments[2], table_name
 
+        warn_deployment_query = session.query(Log).filter(Log.event == "robots").count()
+        if (
+            permissions.ACTION_CAN_ACCESS_MENU,
+            permissions.RESOURCE_ADMIN_MENU,
+        ) in user_permissions and warn_deployment_query > 0:
+            flash(
+                Markup(
+                    'Recent requests have been made to /robots.txt. '
+                    'This indicates that this deployment may be accessible to the public internet. '
+                    'This warning can be disabled by setting webserver.warn_deployment_exposure=False in '
```

Comment:
Can this warning be disabled? I don't see the value of `warn_deployment_exposure` actually being referenced anywhere other than the config.

Reply:
removed it during a merge, I will put it back